polonius-engine/src/output/mod.rs

```diff
@@ -91,6 +91,7 @@ pub struct Output<T: FactTypes> {
     pub var_drop_live_at: FxHashMap<T::Point, Vec<T::Variable>>,
     pub path_maybe_initialized_at: FxHashMap<T::Point, Vec<T::Path>>,
     pub var_maybe_initialized_on_exit: FxHashMap<T::Point, Vec<T::Variable>>,
+    pub known_subset: FxHashMap<T::Origin, BTreeSet<T::Origin>>,
```

Comment:
this doesn't seem useful in the new formulation, but maybe `known_contains`?

Reply:
done